### PR TITLE
Review project from trials

### PR DESCRIPTION
**Pull request by butlermuwo**
## Review project from trials
In this code I was reviewing a project I did during the trials. 

- it is always advisable to put the `script` file just before the `body` closing tag. [Make reference here](https://github.com/butlermuwo/project-from-trials/blob/b0101704cabb80a84bc1c08b137c34299cd5c562/index.html#L10)
- It's good practice to add the viewpoint for responsive web design in the `head` tag. [Make reference here] (https://github.com/butlermuwo/project-from-trials/blob/b0101704cabb80a84bc1c08b137c34299cd5c562/index.html#L3)
- It is advisable to add the `alt` attribute inside your `image` tag for accessibility purposes, allowing for a user to get a full description of the `image` in an event the `image` doesn't load. [Make reference here] (https://github.com/butlermuwo/project-from-trials/blob/b0101704cabb80a84bc1c08b137c34299cd5c562/index.html#L12)
- frameBorder="0" is obsolete and it's best practice to use CSS instead  😃. [Make reference here] (https://github.com/butlermuwo/project-from-trials/blob/b0101704cabb80a84bc1c08b137c34299cd5c562/index.html#L25)
- There is a duplicate of `border` in this property, it best practice not to duplicate values.  [Make reference  here] (https://github.com/butlermuwo/project-from-trials/blob/b0101704cabb80a84bc1c08b137c34299cd5c562/css/style.css#L60)

### CODE REVIEW

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @butlermuwo , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️

- [ ] Kindly use inline comments for your review. An inline comment is a feature in GitHub that allows reviewers to bring an author's attention to some specific lines of codes that require changes. You can read more on inline comments in this [article](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request)



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/style.css [hunk 1]
```diff
@@ -60,7 +61,6 @@ form {
 #email {
   padding: 10px;
   margin-top: 20px;
-  border: none;
```
Comment (butlermuwo) [inline css/style.css:63]: - There is a duplicate of `border` in this property, it best practice not to duplicate values. There is a duplicate of `border` in this property, it best practice not to duplicate values.

File: index.html [hunk 2]
```diff
@@ -2,16 +2,16 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <title>Fork Me! FCC: Test Suite Template</title>
-    <link rel="stylesheet" href="css/style.css" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Fork Me FCC: Test Suite Template</title>
```
Comment (butlermuwo) [inline index.html:7]: - It's good practice to add the viewpoint for responsive web design across different devices in the `head` tag. 😃

File: index.html [hunk 3]
```diff
@@ -2,16 +2,16 @@
...
   </head>
-
```
Comment (butlermuwo) [inline index.html:8]: - It is advisable to remove all blank lines without any code, use extensions in your code editor to help you with formatting lines of code, for easy work efficiency. 👍

File: index.html [hunk 4]
```diff
@@ -2,16 +2,16 @@
...
   <body>
     <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
     <header id="header">
       <img
         id="header-img"
-        src="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png"
+        src="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png" alt=""
```
Comment (butlermuwo) [inline index.html:14]: - It is a requirement to add the alt attribute inside your `image` tag for accessibility purposes, allowing a user to get a full description of the `image` in an event that the `image` doesn't load.

File: index.html [hunk 5]
```diff
@@ -28,7 +28,6 @@ <h1>Handcrafted, home-made masterpieces</h1>
       height="315"
       src="https://www.youtube.com/embed/y8Yv4pnO7qc"
       title="YouTube video player"
-      frameborder="0"
```
Comment (butlermuwo) [inline index.html:31]: - frameBorder="0" is obsolete and it's best practice to use `CSS` instead. 😃

**Review by butlermuwo — COMMENTED** [review]
- Changes to be made ♻️

**Review by brytebee — APPROVED** [review]
Hi @butlermuwo 👋,

# STATUS: APPROVED 🕺🥳 

Congratulations🍾🥳!! Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 



Good luck! 🎉 

- [x] **Optional** Always add your node modules to the `.gitignore` file. Leaving the node module is unprofessional and negates clean code. 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by brytebee — APPROVED** [review]
Hi @butlermuwo 👋,

# STATUS: APPROVED 🕺🥳 

Congratulations🍾🥳!! Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 



Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by butlermuwo** [comment]
- closed pull request.

**Pull request closed without merging**